recipe/0002-bypass-fmath-non-intel-cpus.patch

```diff
@@ -0,0 +1,43 @@
+--- a/include/galsim/fmath/fmath.hpp
++++ b/include/galsim/fmath/fmath.hpp
+@@ -33,6 +33,11 @@
```

Comment:
@rmjarvis what are your thoughts on the performance hit from this patch?

Reply:
He already approved it on galsim/main so ... not devastating?

Reply:
Ohh ok. Nvm then

Reply:
Right.  The fmath stuff requires SSE commands.  So if you have something that can't do SSE, then this just prevents it from breaking on compile.  In my experience the fmath stuff is like a 20-30% speed up for typical use cases, so that's about how much slower this will be than it could be if someone did the work to specialize the fast exp to systems that can't use SSE.  But that is presumably a fair bit of work to figure out.